From: Christian Marangi <ansuelsmth@gmail.com>
Subject: kernel/mtd: add support for EON EN25QX128A

Add support for EON EN25QX128A with no flags as it does
support SFDP parsing.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 drivers/mtd/spi-nor/spi-nor.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/mtd/spi-nor/eon.c
+++ b/drivers/mtd/spi-nor/eon.c
@@ -29,6 +29,7 @@ static const struct flash_info eon_nor_p
 	{ "en25qh128",  INFO(0x1c7018, 0, 64 * 1024,  256) },
 	{ "en25qh256",  INFO(0x1c7019, 0, 64 * 1024,  512)
 		PARSE_SFDP },
+	{ "en25qx128a", INFO(0x1c7118, 0, 64 * 1024,  256) },
 	{ "en25s64",	INFO(0x1c3817, 0, 64 * 1024,  128)
 		NO_SFDP_FLAGS(SECT_4K) },
 };
